pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/filter/query/BaseAbstractQueryBuilder.java

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsm.model.elastic.filter.query;
+
+import java.util.List;
+import java.util.Map;
+
+import org.broadinstitute.dsm.model.Filter;
+import org.broadinstitute.dsm.model.elastic.export.generate.PropertyInfo;
+import org.broadinstitute.dsm.model.elastic.export.parse.Parser;
+import org.broadinstitute.dsm.model.elastic.filter.AndOrFilterSeparator;
+import org.broadinstitute.dsm.model.elastic.filter.FilterParser;
+import org.broadinstitute.dsm.model.elastic.filter.FilterStrategy;
+import org.broadinstitute.dsm.model.elastic.filter.Operator;
+import org.broadinstitute.dsm.model.elastic.filter.splitter.SplitterStrategy;
+import org.elasticsearch.index.query.AbstractQueryBuilder;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.index.query.QueryBuilder;
+
+public class BaseAbstractQueryBuilder {
+    protected String filter;
+    protected Parser parser;
+    protected String esIndex;
+    protected BoolQueryBuilder boolQueryBuilder;
+    protected QueryBuilder queryBuilder;
+    protected SplitterStrategy splitter;
+    protected AndOrFilterSeparator filterSeparator;
+    private BaseQueryBuilder baseQueryBuilder;
+
+    protected BaseAbstractQueryBuilder() {
+        boolQueryBuilder = new BoolQueryBuilder();
+        parser = new FilterParser();
+    }
+
+    public void setEsIndex(String index) {
```

Comment:
we could also remove the ES here like we did for the object classes. there are only ES indexes…

Reply:
do you mean to remove this method?